### ISSUE TITLE
[Back to Bed] HDR retrofit causes a black screen

### ISSUE THREAD

**Issue by Silent-Remux** [SpecialKO/SpecialK#16]
Enabing HDR in this Unity 5 title results in a black screen in game but not in the menus. 
No remastering options used. 
SK version: 22.7.19

Menus (hdr working correctly):
![Back to Bed 23_07_2022 14_08_08](https://user-images.githubusercontent.com/16107603/180606446-008d2074-40e8-47f0-b8bc-94f074dab074.png)

In-game HDR:
![Back to Bed 23_07_2022 14_08_36](https://user-images.githubusercontent.com/16107603/180606462-7f106bf8-3903-4643-a3c9-b55c953fc3aa.png)

In-game SDR:
![🔆hdr - Discord 23_07_2022 14_17_43](https://user-images.githubusercontent.com/16107603/180606751-934c49ef-a90f-4023-bd4d-4b0d6339d7ec.png)

**Comment by Silent-Remux** [comment]
[SpecialK.log](https://github.com/SpecialKO/SpecialK/files/9173849/SpecialK.log)

**Comment by Aemony** [comment]
Have you tried with any of the remastering options enabled?

**Comment by Aemony** [comment]
Follow-up question -- did you try restarting the game as well, or generally tweak the HDR settings? Because everything seems to work fine in my testing after a restart of the game:

![Back to Bed 2022-07-23 16_37_56 (1080p)](https://user-images.githubusercontent.com/10578344/180609973-b624f5f6-a628-4296-b0eb-7c56964729a1.png)

**Comment by Silent-Remux** [comment]
> Follow-up question -- did you try restarting the game as well, or generally tweak the HDR settings? Because everything seems to work fine in my testing after a restart of the game:
> 
> ![Back to Bed 2022-07-23 16_37_56 (1080p)](https://user-images.githubusercontent.com/10578344/180609973-b624f5f6-a628-4296-b0eb-7c56964729a1.png)

I always restart the game after making any changes. 

Here is my global default_SpecialK.ini
```
[Textures.D3D11]
Cache=false

[Input.Gamepad]
AllowHapticUI=false

[Render.FrameRate]
TargetFPS=120.0

[Render.OSD]
HDRLuminance=0.9

[Window.System]
Borderless=true
Center=true
RenderInBackground=true
AlwaysOnTop=2

[SpecialK.HDR]
Use10BitSwapChain=false
Use16BitSwapChain=true
Promote8BitRTsTo16=false
Promote10BitRTsTo16=false
Promote11BitRTsTo16=false
AllowFullLuminance=true
sRGBBypassMode=0
Preset=3
scRGBLuminance_[0]=11.25
scRGBPaperWhite_[0]=5.625
scRGBGamma_[0]=1.85
ToneMapper_[0]=0
Saturation_[0]=1.0
MiddleGray_[0]=1.25
PerceptualBoost0_[0]=-7.0
PerceptualBoost1_[0]=9.0
PerceptualBoost2_[0]=1.1333
PerceptualBoost3_[0]=0.666
scRGBLuminance_[1]=3.4625
scRGBPaperWhite_[1]=2.8125
scRGBGamma_[1]=1.25
ToneMapper_[1]=0
Saturation_[1]=1.075
MiddleGray_[1]=1.25
PerceptualBoost0_[1]=9.05
PerceptualBoost1_[1]=12.195
PerceptualBoost2_[1]=1.06
PerceptualBoost3_[1]=0.83
scRGBLuminance_[2]=1.0
scRGBPaperWhite_[2]=1.0
scRGBGamma_[2]=1.0
ToneMapper_[2]=2
Saturation_[2]=1.0
MiddleGray_[2]=1.25
PerceptualBoost0_[2]=-7.0
PerceptualBoost1_[2]=9.0
PerceptualBoost2_[2]=1.1333
PerceptualBoost3_[2]=0.666
scRGBLuminance_[3]=8.5
scRGBPaperWhite_[3]=1.0
scRGBGamma_[3]=1.0
ToneMapper_[3]=0
Saturation_[3]=1.0
MiddleGray_[3]=1.25
PerceptualBoost0_[3]=20.0
PerceptualBoost1_[3]=3.0
PerceptualBoost2_[3]=1.5
PerceptualBoost3_[3]=1.5
AdaptiveToneMap=false
Enable128BitPipeline=false
```

**Comment by Aemony** [comment]
Having tested this with K-SILENT v1.0 on Discord, I'm clueless as to what might be causing this. I believe however I've seen multiple users over the last couple of months report that SK's HDR retrofit cause a black screen in some games and on some systems, but it's really random. For K-SILENT this game has the issue on Win11, meanwhile I don't experience it at all on Win11.

For other games I've heard some experiences an issue only on Win10, and for others only on Win11...

Driver issue perhaps? Something else that differs between the systems?

I'll leave this ticket open to allow us to track it and possibly gather more data of affected games and OSes.

**Comment by Silent-Remux** [comment]
Fixed.

TLDR: Unity does not like you messing with backbuffers. Used the following and managed to get HDR working again with no black screens. Used the following config and it worked (rename to ini)
 
[SpecialK.txt](https://github.com/SpecialKO/SpecialK/files/9174148/SpecialK.txt)
